OpenTabletDriver.UX/Controls/Generic/Text/Providers/RegexBasedTextProvider.cs

```diff
@@ -0,0 +1,17 @@
+using System.Text.RegularExpressions;
+
+namespace OpenTabletDriver.UX.Controls.Generic.Text.Providers
+{
+    public abstract class RegexBasedTextProvider<T> : MaskedTextProvider<T>
```

Comment:
Would prefer this class name
```suggestion
    public abstract class RegexTextProvider<T> : MaskedTextProvider<T>
```